packages/react-compass/src/theme/theme.ts

```diff
@@ -239,6 +240,8 @@ export const lightTheme = defineTheme({
     gradientOrange: '#F02627, #EA8816',
     gradientRainbow: '#0163AE, #D80837, #53AF35',
     gradientOrangeBlue: '#D2132A, #0597DB',
+
+    secondaryGreen: '#159A00',
```

Comment:
```suggestion
    greenShades: '#159A00',
```
I think should be add suffixes `Shades` for color of h5

Reply:
ref: https://github.com/comfortdelgro/compass-design/blob/145241ba1ae205ac714aaa89d365b2637e83e130/packages/react-compass/src/theme/theme.ts#L181

Reply:
I have changed the color name, thank you @hoapn-fsoft